src/Ec2MetadataGetter.php

```diff
@@ -80,6 +79,88 @@ class Ec2MetadataGetter
     const NOT_AVAILABLE = 'not available';
 
     /**
+     * Whether to return dummy data or not
+     * @var boolean
+     */
+    private $dummy = false;
+
+    public function __construct($cache_dir = "/tmp")
+    {
+        $this->cache_dir = $cache_dir;
```

Comment:
`$this->cache_dir` looks like undefined in Ec2MetadataGetter.
So, `$this->cache_dir`  can be rewritten freely.

```
[root@b517f2809e82 ec2-metadata]# cat a.php 
<?php
require_once 'vendor/autoload.php';
$client = new \Razorpay\EC2Metadata\Ec2MetadataGetter();
$client->cache_dir = 'hoge';
echo $client->cache_dir . PHP_EOL;

[root@b517f2809e82 ec2-metadata]# php a.php 
hoge
[root@b517f2809e82 ec2-metadata]# 

```
    
I think we need the declaration of private variables `$cache_dir`.
And, I prefer the name of the `$cacheDir` than `$cache_dir`.Don't you think so?

e.g.
```php
     /**
      * request is cached in this directory.
      *
      * @var string
      */
     private $cacheDir;
```